package.json

```diff
@@ -9,20 +9,21 @@
 		"webpack": "webpack --mode=production"
 	},
 	"license": "MIT",
-	"dependencies": {},
+	"peerDependencies": {
```

Comment:
Should (at least) these not be `optionalPeerDependencies`?
If the user does not intend to use the apex-chart component, why should they download the apex-chart dependency?